common/lib/xmodule/xmodule/tests/test_video.py

```diff
@@ -42,6 +101,7 @@ def instantiate_descriptor(**field_data):
 
 
 class VideoModuleTest(LogicTest):
+
```

Comment:
I just ran pylint locally on a test file, and I didn't see any warning about not enough whitespace. Pylint has always had rules about 1 vs 2 blank lines -- you need 2 lines between class definitions, and 1 line between method definitions. However, you don't need this extra whitespace between naming the class and writing the docstring, and it fact it makes it *harder* to read. Please remove the whitespace, and if pylint errors show up as a result, let me know.